src/components/footer/footer.tsx

```diff
@@ -1,19 +1,84 @@
 'use client';
 
-import Box from '@mui/material/Box';
-import Typography from '@mui/material/Typography';
-import * as React from 'react';
+import { Box, Button, Container, IconButton, Link } from '@mui/material';
+import Image from 'next/image';
 import './footer.scss';
 
 function Footer() {
   return (
-    <Box className="footer-container" component="footer">
-      <Typography variant="h6" className="footer-title" gutterBottom>
-        OpenBC
-      </Typography>
-      <Typography variant="subtitle1" className="footer-subtitle" component="p">
-        Sample Text
-      </Typography>
+    <Box component="footer" className="footer-container">
+      <Container className="footer">
+        <Box className="footer__wrapper">
+          <Box className="footer__menu">
+            <Link href="/" className="footer__logo">
+              OpenBC
+            </Link>
+            <Box className="footer__buttons">
+              <Link href="/about" className="footer__button">
+                About
+              </Link>
+              <Link
+                href="/join-us"
+                className="footer__button footer__button--highlight"
+              >
+                Join Us!
```

Comment:
It may need to change the blue color to grey to match with Figma design. Otherwise, the rest is okay.